validate/validators/bam.py

```diff
@@ -27,7 +28,8 @@ def check_bam_index(path):
 
     return True
 
-def check_bam(path:Path):
+# pylint: disable=W0613
+def check_bam(path:Path, args:Union[argparse.Namespace,None]):
```

Comment:
Is there a more specific type, or a wider set of union types that `args` can be? If we can specify a type that fits all functions where this is relevant that might help us manage complexity.

Reply:
I think we can have two types for `args`: `ChecksumArgs` and `ValidateArgs`, I'm updating the `CramArgs` to be `ValidateArgs`, which can be used for all functions calls downstream of `run_validate`